features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/modules/device.js

```diff
@@ -332,5 +334,75 @@ deviceModule = function () {
         return license;
     };
 
+    publicMethods.getOwnDevices = function () {
+        devicemgtService = devicemgtProps["httpsURL"] + "/common/device_manager";
+        listAllDevicesEndPoint = devicemgtService + "/device/user/" + user.username + "/all";
+        var result = get(listAllDevicesEndPoint, {}, "json");
+        var devices = result.data;
+        var device;
+        for (var d in devices){
```

Comment:
Acknowledge, Already fixed in moved code with commit https://github.com/wso2/carbon-device-mgt/commit/e537e2d841ca3c7946cd34e9da96f261e0ac94e1